src/core/on_internal_error.cc

```diff
@@ -58,3 +58,8 @@ void seastar::on_internal_error_noexcept(logger& logger, std::string_view msg) n
         abort();
     }
 }
+
+void seastar::on_fatal_internal_error(logger& logger, std::string_view msg) noexcept {
+    set_abort_on_internal_error(true);
+    on_internal_error_noexcept(logger, msg);
```

Comment:
Why not just
```c++
    logger.error("{}, at: {}", msg, current_backtrace());
    abort();
```
?

Reply:
can do, I just wanted to use the common code.
maybe we can refactor the logger.error part into a common function that both places can call.

Reply:
It's common code but obfuscated.